modules/ROOT/pages/installation.adoc

```diff
@@ -7,3 +7,472 @@ endif::[]
 Install API Community Manager with these instructions.
 
 Installed as a Salesforce package.
+
+API Community Manager 
+Setup Guide
+ (RC Version)
+
+Organization Installation Requirements - prereqs???
+Installing API Community Manager 
+Update API Community Manager - move this???
+Create a Community
+API Community Manager Components
```

Comment:
this is already on a different section of docs - you can take it out here

Reply:
Removed - thank you!